src/pymedphys/cli/dicom.py

```diff
@@ -81,16 +84,81 @@ def dicom_adjust_rel_elec_density(dicom_subparsers):
     parser.set_defaults(func=adjust_RED_cli)
 
 
-def dicom_structure_name_RED_adjust(dicom_subparsers):
+def adjust_RED_by_structure_name(dicom_subparsers):
     parser = dicom_subparsers.add_parser(
         'adjust-RED-by-structure-name',
-        help=(
-            'Use structure naming conventions to automatically adjust the'
-            'DICOM RED in the header. For example, naming a structure '
-            '``a_structure_name RED=1.15`` will cause that structure to have '
-            'an override of 1.15 applied.'))
+        help=('Use structure naming conventions to automatically '
+              'adjust the relative electron density of a structure '
+              'within a DICOM RT Structure set. For example, naming '
+              'a structure ``a_structure_name RED=1.15`` will cause '
+              'that structure to have an override of 1.15 applied.'))
 
     parser.add_argument('input_file', type=str, help='input_file')
     parser.add_argument('output_file', type=str, help='output_file')
 
     parser.set_defaults(func=adjust_RED_by_structure_name_cli)
+
+
+def anonymise(dicom_subparsers):
+    parser = dicom_subparsers.add_parser(
+        'anonymise',
+        help=("Anonymise DICOM files."))
+
+    parser.add_argument(
+        'input_path',
+        type=str,
+        help=("Input file or directory path. If a directory is "
+              "supplied, all DICOM files within the directory and its "
+              "subdirectories will be anonymised"))
+
+    parser.add_argument(
+        '-d', '--delete_original_files',
+        action='store_true',
+        help=("Use this flag to delete the original, non-anonymised "
+              "files in the processed directory. The original files "
+              "will only be deleted if anonymisation completes "
+              "successfully."))
```

Comment:
I think it's worth considering how to fail gracefully. For example if the directory is half done then there are a heap of files left in the directory already anonymised, and a heap not anonymised. Rerunning the code in its current form I believe will then try and anonymise everything, including those that have been anonymised already.

Maybe this isn't a problem... But maybe it's worth considering.

Reply:
See latest commit :)